src/style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.box {
+  height: 300px;
+  min-width: 300px;
+  flex-basis: 33.33%;
+  flex-shrink: 0;
+  background-color: red;
+  flex-grow: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #fff;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.box:nth-child(1) {
+  background-color: rgb(100%, 0%, 0%);
+}
+
+.box:nth-child(2) {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.box:nth-child(3) {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.box:nth-child(4) {
+  background-color: rgb(40%, 0%, 0%);
+  order: 2;
+}
+
+.box:nth-child(5) {
+  background-color: rgb(20%, 0%, 0%);
+  order: 1;
+}
+
+.box:nth-child(6) {
+  background-color: rgb(0, 0, 0);
+}
+
+@media (max-width: 899px) {
+  .box {
+    flex-basis: 50%;
```

Comment:
This is what I see on 600px breakpoint now, fix it
![image](https://user-images.githubusercontent.com/37161315/136962113-42395466-748f-411d-8eb7-9397797d90ee.png)